worlds/oot/__init__.py

```diff
@@ -1271,9 +1301,17 @@ def get_region(self, region_name):
             self._regions_cache[region_name] = ret
             return ret
 
+    @functools.cache
+    def get_entrances(self):
```

Comment:
self.multiworld.get_entrances(self.player) gives you all entrances for that player, already cached and already filtered.